ReactiveCocoa/Swift/Property.swift

```diff
@@ -355,61 +325,61 @@ public struct AnyProperty<Value>: PropertyType {
 	}
 
-	/// Initializes a property that first takes on `initialValue`, then each value
+	/// Initializes a property that first takes on `initial`, then each value
 	/// sent on `signal`.
-	public init(initialValue: Value, signal: Signal<Value, NoError>) {
-		self.init(propertyProducer: SignalProducer(signal: signal).prefix(value: initialValue),
+	public init(initial: Value, followingBy signal: Signal<Value, NoError>) {
+		self.init(sourcingFrom: SignalProducer(signal: signal).prefix(value: initial),
 		          capturing: [])
 	}
 
 	/// Initializes a property by applying the unary `SignalProducer` transform on
 	/// `property`. The resulting property captures `property`.
-	private init<P: PropertyType>(_ property: P, @noescape transform: SignalProducer<P.Value, NoError> -> SignalProducer<Value, NoError>) {
-		self.init(propertyProducer: transform(property.producer),
-		          capturing: AnyProperty.capture(property))
+	private init<P: PropertyProtocol>(sourcingFrom property: P, transform: @noescape (SignalProducer<P.Value, NoError>) -> SignalProducer<Value, NoError>) {
+		self.init(sourcingFrom: transform(property.producer),
+		          capturing: AnyProperty.capturing(property))
 	}
 
 	/// Initializes a property by applying the binary `SignalProducer` transform on
-	/// `property` and `anotherProperty`. The resulting property captures `property`
-	/// and `anotherProperty`.
-	private init<P1: PropertyType, P2: PropertyType>(_ firstProperty: P1, _ secondProperty: P2, @noescape transform: SignalProducer<P1.Value, NoError> -> SignalProducer<P2.Value, NoError> -> SignalProducer<Value, NoError>) {
-		self.init(propertyProducer: transform(firstProperty.producer)(secondProperty.producer),
-		          capturing: AnyProperty.capture(firstProperty) + AnyProperty.capture(secondProperty))
+	/// `properties.first` and `properties.second`. The resulting property captures
+	/// the two property sources.
+	private init<P1: PropertyProtocol, P2: PropertyProtocol>(sourcingFrom properties: (first: P1, second: P2), transform: @noescape (SignalProducer<P1.Value, NoError>) -> (SignalProducer<P2.Value, NoError>) -> SignalProducer<Value, NoError>) {
+		self.init(sourcingFrom: transform(properties.first.producer)(properties.second.producer),
+		          capturing: AnyProperty.capturing(properties.first) + AnyProperty.capturing(properties.second))
 	}
 
 	/// Initializes a property from a producer that promises to send at least one
 	/// value synchronously in its start handler before sending any subsequent event.
 	/// If the producer fails its promise, a fatal error would be raised.
 	///
 	/// The producer and the signal of the created property would complete only
-	/// when the `propertyProducer` completes.
-	private init(propertyProducer: SignalProducer<Value, NoError>, capturing sources: [Any]) {
+	/// when the `unsafeProducer` completes.
+	private init(sourcingFrom unsafeProducer: SignalProducer<Value, NoError>, capturing sources: [Any]) {
```

Comment:
Same here too. I don't think we need to worry about updating `private` APIs at all in this PR.